src/docs/cookbook/navigation/navigation-basics.md

```diff
@@ -94,13 +94,8 @@ onPressed: () {
 
 ## 3. Return to the first screen using `Navigator.pop`
 
-Now that we're on our second screen, how do we close it out and return to the
-first? Using the [`Navigator.pop`](https://docs.flutter.io/flutter/widgets/Navigator/pop.html)
-method! The `pop` method will remove the current `Route` from the stack of
-routes managed by the navigator.
-
-For this part, we'll need to update the `onPressed` callback found in our
-`SecondScreen` Widget
+The [`Navigator.pop`](https://docs.flutter.io/flutter/widgets/Navigator/pop.html)
```

Comment:
The [`Navigator.pop()`](...) method removes the current route from the stack of routes.